generators/python/urllib3/templates/README.mustache

```diff
@@ -0,0 +1,121 @@
+# Onfido Python Library
+
+The official Python library for integrating with the Onfido API.
+
+Documentation can be found at <{{ documentationUrl }}>.
+
+This version uses Onfido API {{ apiVersion }}. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
+
+## Requirements
+
+
+Python {{{generatorLanguageVersion}}}
+
+## Installation & Usage
+### pip install
+
+If the python package is hosted on a repository, you can install directly using:
+
+```sh
+pip install {{{packageName}}}
+```
+
+(you may also use poetry: `poetry add {{{packageName}}}`)
+
+Then import the package:
+```python
+import {{{packageName}}}
+```
+
+### Setuptools
+
+Install via [Setuptools](http://pypi.python.org/pypi/setuptools).
+
+```sh
+python setup.py install --user
+```
+(or `sudo python setup.py install` to install the package for all users)
+
+Then import the package:
+```python
+import {{{packageName}}}
+```
+
+### Tests
+
+Execute `pytest` to run the tests.
+
+## Getting Started
+
+Import the `DefaultApi` object, this is the main object used for interfacing with the API:
+
+```python
+import {{{packageName}}}
+
+import urllib3
+from os import environ
+
+configuration = {{{packageName}}}.Configuration(
+    api_token=environ['ONFIDO_API_TOKEN'],
+    region={{{packageName}}}.configuration.Region.EU,     # Supports `EU`, `US` and `CA`
+    timeout=urllib3.util.Timeout(connect=60.0, read=60.0)
+  )
+
+with {{{packageName}}}.ApiClient(configuration) as api_client:
+  onfido_api = onfido.DefaultApi(api_client)
+  ...
+```
+
+NB: by default, timeout values are set to 30 seconds.
+
+## Making a call to the API
+
+```python
+  try:
```

Comment:
Might be nitpicking (and we have done the same in the other READMEs) but the snippet of webhook verification shouldn't be in the same place as when an applicant is created because it happens in different times in the lifecycle of a check/workflow run

Reply:
Yeah, good point. I haven't put it aside because I didn't want to make README files longer, but probably it makes sense putting that aside also for clarity.

Reply:
[Updated](https://github.com/onfido/onfido-openapi-spec/pull/64/commits/080d396e09c2350d4035ea1b2274c8c0881aec8c). I took the opportunity also align a little bit contents and style of the different README.md files.